Processes/DevRel-Publication-Process.md

```diff
@@ -0,0 +1,169 @@
+
+**Note**: New or updated content is published to a private staging environment every day, which allows for content review prior to publication of content to the public dev portal. Once content is reviewed and approved for public consumption, the content can be published to the public dev portal. This publication process can be scheduled or ad hoc, depending on the need for the community to consume the content.
+
+### When to Use this Procedure
+Use this procedure when you need to update content from multiple repositories to the [EOS Dev Portal](http://docs.eosnetwork.com).
+
+### Expected Results
+Fresh, updated content is available for consumption on the EOS Dev Portal.
+
+# Procedures Contained in This Publication Process
+Complete the following procedures to publish new or updated content to the EOS Dev Portal:
+1. Create/Edit Files
+2. Create Commits/Push Changes
+3. Check Changes on Your Remote/Target Branch
+4. Pull Remote Changes from Your Remote Branch
+5. Update Existing Files (Refer to Create/Edit Files)
+6. Rename Files
+
+## Definitions
+The following table provides definitions for Github branches used in this process:
+
+***Note***: A branch is a parallel version of a repository. It is contained within the repository, but does not affect the primary or main branch allowing you to work freely without disrupting the “live” version. When you’ve made the changes you want to make, you can merge your branch back into the main branch to publish your changes. ([GitHub Docs: Glossary](https://docs.github.com/en/get-started/quickstart/github-glossary))
+
+| **Branch** | **Definition** | 
+| --- | --- |
+| Current | The current branch is the present branch. This branch has focus. | 
+| Default | Each repository contains one default branch, which is typically called main. GitHub creates this branch when a repository is initialized. | 
+| Local | The local branch resides in your local environment. Changes made to the local branch are merged into a target branch. | 
+| Main | The main branch is typically the default branch. PRs merge a source branch to the main branch. | 
+| Remote | The remote branch is typically your local branch. | 
+| Source | The source branch contains updates to be merged into the target branch. Typically the changes you made previously to the target branch become the source branch for the final merge into the main branch. | 
+| Target | The repository and branch location where you want your new or updated content to reside | 
+
+## How to Publish New or Updated Content to the EOS Dev Portal
+
+### Before You Begin
+Ensure you have the following items ready before you begin this procedure:
+* Infrastructure - appropriate repository(ies) available for PR/Commit
+* Access to Github Engineering and DevRel repositories, as necessary
+* Create a DevRel ticket/issue - have the issue number (the #N number) available
```

Comment:
Do we want to ALWAYS have a ticket? 

If someone wants to come and create a PR to fix a single typo, I don't want to add unnecessary process on top of that menial task

Reply:
Done

Reply:
Regarding the following comment - This process may be better suited to the Best Practices folder or Onboarding folder